dist/init/freebsd/caddy

```diff
@@ -62,7 +62,7 @@ fi
 pidfile="/var/run/${name}.pid"
 procname="${caddy_bin_path}" #enabled builtin pid checking for start / stop
 command="/usr/sbin/daemon"
-command_args="-u ${caddy_user} -p ${pidfile} /usr/bin/env ${caddy_env} ${procname} -cpu ${caddy_cpu} -log stdout -conf ${caddy_config_path} -agree -email ${caddy_cert_email} < /dev/null >> ${caddy_logfile} 2>&1"
+command_args="-p ${pidfile} /usr/bin/env ${caddy_env} ${procname} -cpu ${caddy_cpu} -log stdout -conf ${caddy_config_path} -agree -email ${caddy_cert_email} < /dev/null >> ${caddy_logfile} 2>&1"
```

Comment:
Wouldn't it be better to change the caddy_user value? Or remove the unused variable then?

Reply:
No. The ${name}_user variable (${name} is caddy here) is special and tells the init system to run the command as the specified user. A documentation can be found in the `${name}_user` part of [man rc.subr][1]. Sorry for being unclear!

[1]: https://www.freebsd.org/cgi/man.cgi?query=rc.subr&sektion=8